datafusion/core/src/physical_plan/aggregates/mod.rs

```diff
@@ -90,8 +90,8 @@ pub struct PhysicalGroupBy {
     null_expr: Vec<(Arc<dyn PhysicalExpr>, String)>,
     /// Null mask for each group in this grouping set. Each group is
     /// composed of either one of the group expressions in expr or a null
-    /// expression in null_expr. If groups[i][j] is true, then the the
-    /// j-th expression in the i-th group is NULL, otherwise it is expr[j].
+    /// expression in null_expr. If `groups[i][j]` is true, then the the
```

Comment:
this is a driveby cleanup -- `rustdoc` was complaining that `i` and `j` were not valid links 😆